packages/storybook/stories/va-accordion-uswds.stories.tsx

```diff
@@ -98,7 +98,7 @@ export default {
   },
 };
 
-const Template = args => {
+const AccordionComponent = memo(args => {
```

Comment:
> [!note]
> Updated this because I was getting this error on the initial load of accordion docs: "Error: Rendered more hooks than during the previous render"